src/LineageTree/export_csv.py

```diff
@@ -0,0 +1,61 @@
+import lineageTree
```

Comment:
Is that working or is it rather
```python
from lineageTree import LineageTree
```
?

Reply:
Yes. This would work as well.
I changed in the PR.

It has to be a small `l` though: 
`from lineageTree import lineageTree`

Reply:
I think it is actually:
```python
from LineageTree import lineageTree
```

Maybe I should change that 😅

Reply:
That strangly gives me an error message: `Unresolved reference 'LineageTree'`
But I am not a Python expert..

Reply:
My feeling is that since both the python file (`lineageTree.py`) and the class within the python file ('class lineageTree') are written with a small `l`, it needs to be small `l`.
The fact the folder has a big `L` does not matter in this case, since both the `lineageTree.py` and the `export_csv.py` live in the same folder.

Reply:
The whole idea was the following:
- `LineageTree` is the library
- `lineageTree` is the class
But that's clearly brings confusion (at least to me).

About your error I am not sure why you get it but in general if you want to refer to the file `lineageTree.py` rather than the library `LineageTree` you should put a `.` in front of the name of the file:
```python
from .lineageTree import lineageTree
```
That would be a relative import.

That does not explain your error though 🧐

Reply:
Thanks for the clarification. I am really not a Python expert (I live more in the Java world).